src/main/java/org/bytedeco/javacv/FrameRecorder.java

```diff
@@ -122,6 +123,7 @@ public static FrameRecorder create(String className, String filename, int width,
     protected int maxBFrames = -1;
     protected int trellis = -1;
     protected int maxDelay = -1;
+    protected Charset charset = Charset.defaultCharset();
```

Comment:
Let's add getCharset()/setCharset() for this one too.

Looks good otherwise, thanks!

Reply:
My bad, I missed that. ＞﹏＜